package.json

```diff
@@ -36,7 +36,7 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "bleno": "~0.1.13",
+    "bleno": "git+https://github.com/jacobrosenthal/bleno#platform-on-state-change",
```

Comment:
Separating it would be great. The characteristic request handling is still WIP right?